File: hammer-gravity/src/main/java/dev/vini2003/hammer/gravity/registry/common/HGEvents.java
package dev.vini2003.hammer.gravity.registry.common;

import dev.vini2003.hammer.gravity.api.common.manager.GravityManager;
import net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents;

public class HGEvents {
	public static void init() {
		ServerPlayConnectionEvents.JOIN.register(new GravityManager.JoinListener());
	}
}

File: hammer-gravity/src/main/kotlin/dev/vini2003/hammer/gravity/registry/common/HGEvents.kt
package dev.vini2003.hammer.gravity.registry.common

import dev.vini2003.hammer.gravity.common.manager.GravityManager
import net.fabricmc.fabric.api.networking.v1.ServerPlayConnectionEvents

object HGEvents {
	fun init() {
		ServerPlayConnectionEvents.JOIN.register(GravityManager.ServerPlayConnectionEventsJoinListener)
	}
}